hbase-shaded-jersey/pom.xml

```diff
@@ -106,6 +106,33 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <id>unpack</id>
```

Comment:
A better ID here would be something like `relocate-jersy-jdk11-support`.

Should also add a comment about what is going on since future maintainers might not already know about multi-release jars